mkdocs.yml

```diff
@@ -105,6 +105,8 @@ nav:
       - debug: rpc/debug.md
   - configuration.md
   - Integration options: integration-options.md
+  - Guides:
```

Comment:
I suggest moving this section after the specifications because it is more for supplemental use case-specific information, while the specifications are the fundamental documentation that would typically be studied first before seeing how those concepts are applied in the guides.